### PR TITLE
Fix lint issues in vertical pod autoscaller

### PR DESCRIPTION
**Pull request by Fedosin**
#### Which component this PR applies to?

vertical-pod-autoscaler

#### What type of PR is this?

/kind cleanup

#### What this PR does / why we need it:

This is a code refactoring for vertical pod autoscaler aimed to meet current golang code formatting requirements.

#### Does this PR introduce a user-facing change?

No

```release-note
None
```

### CODE REVIEW

**Review by elmiko — COMMENTED** [review]
thanks @Fedosin!
/lgtm

File: vertical-pod-autoscaler/pkg/recommender/input/history/history_provider.go [hunk 1]
```diff
@@ -289,6 +289,10 @@ func (p *prometheusHistoryProvider) GetClusterHistory() (map[model.PodID]*PodHis
 			sort.Slice(samples, func(i, j int) bool { return samples[i].MeasureStart.Before(samples[j].MeasureStart) })
 		}
 	}
-	p.readLastLabels(res, p.config.PodLabelsMetricName)
+	err = p.readLastLabels(res, p.config.PodLabelsMetricName)
+	if err != nil {
```
Comment (jbartosik) [inline vertical-pod-autoscaler/pkg/recommender/input/history/history_provider.go:293]: nit: if err = p.readLastLabels(res, p.config.PodLabelsMetricName); err != nil {

**Review by jbartosik — COMMENTED** [review]
Thanks!

Looks good to me, just one small nit pick.

**Review by jbartosik — APPROVED** [review]
/lgtm
/approve

I'd rather merge this as-is than wait for merge conflict to appear

**Pull request merged**